charts/sdfactory/values.yaml

```diff
@@ -32,7 +32,7 @@ image:
     # -- Set the Image Pull Policy
   pullPolicy: Always
     # -- Image tage is defined in chart appVersion.
-  tag: ""
+  tag: "main"
```

Comment:
Do not use main as the default tag for the image. Leave it blank and set it in the template to the Chart.yaml appVersion